### ISSUE TITLE
showstopper error: "wrong status line..." apt-spy2 fails to run

### ISSUE THREAD

**Issue by hardyoyo** [lagged/apt-spy2#5]
Hi, in testing apt-spy2, I've found that it will fail to run with the following error:

wrong status line: "SSH-2.0-OpenSSH_5.9p1 Debian-5ubuntu1.1"

after running either of the following commands

apt-spy2 fix --launchpad --commit --country=$COUNTRY
apt-spy2 check --launchpad --country=$COUNTRY

On googling the error message, I believe this might be an error message from the Nokogiri library:

http://stackoverflow.com/questions/8269904/why-do-i-get-wrong-status-line-errors-from-nokogiri

I will see if I can find the place where the code in apt-spy2 is using Nokogiri and see if I can just get it to understand that such an error is simply a bad response, and handle that case appropriately, instead of merely dying.

**Comment by till** [comment]
@hardyoyo @jcowhigjr apparently I missed these notifications. Any idea if either of you came up with a solution?

**Comment by till** [comment]
@hardyoyo From reading the stack overflow, it seems to be directly related to the "target". Can't say that this is not a bug in my code (or nokogiri) without seeing more output.

**Comment by hardyoyo** [comment]
Hi, sorry, I *also* missed this notification on my end. Yes, my workaround has been to avoid using the --launchpad --country command line options.

**Comment by till** [comment]
@hardyoyo :D That's fine! It doesn't seem like a super-blocker! Is this still an issue, and can you paste more output? I'll see if I can fix it and do a new release.

**Comment by till** [comment]
I merged a bunch of PRs, etc.:

```
❯ bundle exec apt-spy2 check --launchpad --country=us
Mirror: http://mirror.enzu.com/ubuntu/ - UP
Mirror: http://mirror.us.leaseweb.net/ubuntu/ - UP
Mirror: http://mirrors.rit.edu/ubuntu/ - UP
Mirror: http://mirrors.wikimedia.org/ubuntu/ - UP
Mirror: http://ubuntu.osuosl.org/ubuntu/ - UP
Mirror: http://mirror.clarkson.edu/ubuntu/ - UP
Mirror: http://mirror.steadfastnet.com/ubuntu/ - UP
Mirror: http://mirror.ubuntu.serverforge.org/ - UP
```

Having a hard time reproduce this. I am happy to write the code if anyone has more output.

**Comment by till** [comment]
Released v0.7.0 (changelog: #23):
https://rubygems.org/gems/apt-spy2
